test/js/web/fetch/wasm-streaming.test.ts

```diff
@@ -0,0 +1,239 @@
+import { describe, expect, test } from "bun:test";
```

Comment:
more tests in https://github.com/web-platform-tests/wpt/blob/master/wasm/webapi/wasm_stream_compile_test.html if you were curious